hudi-spark/src/test/java/org/apache/hudi/client/TestBootstrap.java

```diff
@@ -770,7 +770,8 @@ public TestRandomBootstapModeSelector(HoodieWriteConfig writeConfig) {
   }
 
   HoodieWriteConfig.Builder getConfigBuilder(String schemaStr) {
-    HoodieWriteConfig.Builder builder = getConfigBuilder(schemaStr, IndexType.BLOOM);
+    HoodieWriteConfig.Builder builder = getConfigBuilder(schemaStr, IndexType.BLOOM)
+        .withExternalSchemaTrasformation(true);
```

Comment:
If we set this to false, you will see that full bootstrap related tests will start to fail  with schema compatibility issues between parquet reader generated avro records and writer schema

Reply:
@umehrot2  : Added a note here